### PR TITLE
Counts more stuff as suicide

### PR DESCRIPTION
**Pull request by MatheusIsNotAvailable**
[BALANCE]

## About the PR 
 Makes it so more methods of killing yourself count as suicide, I didn't touch some stuff on porpouse as those require more effort than going to the bar and grabbing a russian revolver or going to the chapel and grabbing a bible, if it should all count just tell me.
## Why's this needed? 
Being able to easily blow up somewhere with no risk of it not working by using micro/macro bombs and something easily obtainable is bad.

### CODE REVIEW

**Comment by Camryn-Buttes** [comment]
Good idea, but I do have an issue with the bible fart being classified as a suicide since there are multiple ways to force someone to fart on a bible or to trick them into farting on it (there's even a miniature bible whose smaller sprite is specifically intended to facilitate this trickery)

**Comment by penny64** [comment]
I don't think this makes any sense, the only time there's metaphysical effects on your death is when you use the 'suicide' verb.
It's only expected that the rules change when you activate this function.
 How is a player to know that a Bible is or isn't "suicide"?
Or a Russian revolver? 
When a new user implants themselves with a micromb and grabs a Russian revolver, how will they understand why they died on accident but didn't explode comedically?
How does a player know a bomb won't just count as suicide if this contract is broken?
Mechanically all of these should work until you press the button that changes the rules, this breaks the contract

File: code/obj/item/gun/russian.dm [hunk 1]
```diff
@@ -47,8 +47,12 @@
 			playsound(user, "sound/weapons/Gunshot.ogg", 100, 1)
 			for(var/mob/O in AIviewers(user, null))
 				if (O.client)	O.show_message("<span class='alert'><B>BOOM!</B> [user]'s head explodes.</span>", 1, "<span class='alert'>You hear someone's head explode.</span>", 2)
+				user.suiciding = 1
 				user.TakeDamage("head", 300, 0)
 				take_bleeding_damage(user, null, 500, DAMAGE_STAB)
+				SPAWN_DBG(50 SECONDS)
```
Comment (PrimeNumb) [inline code/obj/item/gun/russian.dm:53]: Why 50 seconds?

**Comment by PrimeNumb** [comment]
> I don't think this makes any sense, the only time there's metaphysical effects on your death is when you use the 'suicide' verb.
> It's only expected that the rules change when you activate this function.
> How is a player to know that a Bible is or isn't "suicide"?
> Or a Russian revolver?
> When a new user implants themselves with a micromb and grabs a Russian revolver, how will they understand why they died on accident but didn't explode comedically?
> How does a player know a bomb won't just count as suicide?
> Mechanically all of these should work until you press the button that changes the rules, this breaks the contract

Agreed, this change would make it very unintuitive and ambiguous.

Also, please be a little more descriptive in the PR, i.e. list every suicide that you touched. This makes it so people can quickly get a better grasp of what your PR actually changes, without having to poke around in the code to figure out what you did. Poking around in the code is better suited for reviewing _how_ you implemented your changes rather than _what_ you changed.

**Comment by MatheusIsNotAvailable** [comment]
Penny is right, until I think of some better way to do this, wich I don't think there is, I would rather It not be a thing at all, would generate a lot of confusion.

**Pull request closed without merging**